File: c#/findMax.cs
using System;

// To execute C#, please define "static void Main" on a class
// named Solution.

class Solution
{
    static void Main(string[] args)
    {
        int[] arrays = {1,2,3,4,5};
       /* foreach(int array in arrays)
        {
            Console.WriteLine(array);
        }*/
        
        int maxValue = findMax(arrays);
        Console.WriteLine(maxValue);
/*        for (var i = 0; i < 5; i++)
        {
            Console.WriteLine("Hello, World");
        }*/
    }
    
    public static int findMax(int[] arrays)
    {
        int maxValue = -1;
        foreach(int array in arrays)
        {
            if (array > maxValue)
            {
                maxValue = array;
            }
        }
        return maxValue;
    }
    
}

File: java/findMax.java
import java.io.*;
import java.util.*;

/*
 * To execute Java, please define "static void main" on a class
 * named Solution.
 *
 * If you need more classes, simply define them inline.
 */

class Solution {
  public static void main(String[] args) {
    
    int array[] = {1, 4, 6, 7, 21, 3};
    int maxValue = -1;

    maxValue = findMax(array);
    System.out.println(maxValue);
/*    ArrayList<String> strings = new ArrayList<String>();
    strings.add("Hello, World!");
    strings.add("Welcome to CoderPad.");
    strings.add("This pad is running Java 8.");

    for (String string : strings) {
      System.out.println(string);
    }*/
  }
  
 public static int findMax(int arrays[]){
    int maxValue = -1;
    for (int array: arrays) {
      if (array > maxValue) {
        maxValue = array;        
      }
    }
    return maxValue;
  }
}
